app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,11 +1,61 @@
 import Avatar from "images/avatar.png";
 import * as yup from "yup";
 
+export const TAG_OPTIONS = [
+  {
+    label: "Getting Started",
+    value: "getting_started",
+  },
+  {
+    label: "Onboarding",
+    value: "onboarding",
+  },
+  {
+    label: "User Flow",
+    value: "user_flow",
+  },
+  {
+    label: "UX",
+    value: "ux",
+  },
+  {
+    label: "Bugs",
+    value: "bugs",
+  },
+  {
+    label: "V2",
+    value: "v2",
+  },
+];
+
+export const ASSIGNED_CONTACT_OPTIONS = [
+  {
+    label: "Value One",
+    value: "value1",
+  },
+  {
+    label: "Value Two",
+    value: "value2",
+  },
+  {
+    label: "Value Three",
+    value: "value3",
+  },
+  {
+    label: "Value Four",
+    value: "value4",
+  },
+  {
+    label: "Value Five",
+    value: "value5",
+  },
+];
+
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
   assigned_contact: null,
-  tag: null,
+  tags: null,
```

Comment:
If `tags` is an array, then its initial value should be `[]`, that is an `empty` array.

Reply:
This is resolved.